vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp

```diff
@@ -133,6 +133,7 @@ class TextInputShadowNode : public ShadowNodeBase {
   void SetSelection(int64_t start, int64_t end);
   winrt::Shape FindCaret(xaml::DependencyObject element);
 
+  bool m_initialUpdateComplete = false;
```

Comment:
consider making all these bools bits ( `: 1` ) and putting them all next to each other to improve layout